reactplay/react-play
# [#2PlaysAMonth]: Image Gallery - Create a responsive image gallery by using the free Unsplash API

> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

The project contains use of Unsplash API, which is fetched using axios and then populated in the Photo Component.

Following things have been used in this project:

1. useState
2. useEffect
3. React Form
4. Component Refactoring
5. Material UI icon
6. Axios npm module
7. CSS Grid
8. Infinite scroll functionality(This feature is still not showing the react play app but is working on local file system. I am working on it)

The app is responsive and redirects to the individual images and profiles when clicked on it.



Fixes [#910](https://github.com/reactplay/react-play/issues/910)



## Type of change



# How Has This Been Tested?

To see working of the app repeat the following 
1. Go to the app. Some images are loaded on the screen by default
2. Search for any keyword. The screen will update with new images.
3. Hover over any image, the cursor will turn to pointer.
4. On clicking any image, the page will re direct to that image.
5. On hovering, the profile name, likes will come in front of the image. On clicking the profile photo, the profile of the image contributor will open
6. The page on scrolling down should show more images. 

# Checklist:


# Screenshots or example output

![image](https://user-images.githubusercontent.com/94108842/219378139-7481538f-61cc-4523-b6ce-c4777c3cebd4.png)

![image](https://user-images.githubusercontent.com/94108842/219378765-b1ffb971-6dbf-4915-9c4e-bbc4122734dc.png)

![image](https://user-images.githubusercontent.com/94108842/219378957-203a3772-5a5c-4e63-afbf-4e26268f4301.png)
